src/leiningen/hiera.clj

```diff
@@ -132,13 +137,18 @@
         context (merge default-options
                        (:hiera project)
                        {:internal-ns (set (file-namespaces source-files))
-                        :graph (file-deps source-files)})]
-    (rhizome/save-graph
-      (graph-nodes context)
-      (partial adjacent-to context)
-      :vertical? (:vertical context)
-      :node->descriptor (partial render-node context)
-      :node->cluster (partial node-cluster context)
-      :cluster->descriptor (fn [c] {:label c})
-      :filename (:path context))
-    (println "Generated namespace graph in" (:path context))))
+                        :graph (file-deps source-files)})
+        conf {:vertical? (:vertical context)
+              :node->descriptor (partial render-node context)
+              :node->cluster (partial node-cluster context)
+              :cluster->descriptor (fn [c] {:label c})}]
+    (apply (partial rhizome/save-graph
```

Comment:
You actually don't need to use `partial` here, since `apply` will already interpret a call like `(apply f a b [c d])` as `(f a b c d)`.